Reference: Get task by title (#67)
UWC2-APIDOC/to-do-service-public: docs/api/tasks-ref-topic-get-task-by-title.md

---
layout: page
---

# Get task by title

> Returns an array of [`task`](task) objects only specified by the `title` parameter, if it exists.

Reviewer: ```suggestion
Returns an array of [`task`](task) objects with the specified `title` parameter, if any exist.
```

## URL

```shell

> {GET}{server_url}/tasks/{title}

Reviewer: ```suggestion
{GET} {server_url}/tasks?title={title}
```

Check to see if the syntax is correct for the query.

Author: I checked. The syntax is correct.
